gatsby/gatsbysourceplone.rst

```diff
@@ -0,0 +1,62 @@
+gatsby-source-plone
+===================
+
+With the previous sections on source nodes, retrieving data from ``plone.restapi`` and finally using the search traversal method, we have understood how our source-plugin works at base level.
+
+Cheers! That was amazing work.
+
+We have already built this plugin with additional helpful features and functionality to handle all kinds of data, caching and so on in an optimal manner.
+
+So let us try out the `gatsby-source-plone <https://github.com/collective/gatsby-source-plone/>`_ plugin with our `gatsby-starter-plone <https://github.com/collective/gatsby-starter-plone/>`_ to instantly kickstart GatsbyJS development with Plone.
```

Comment:
We are talking about source-plone and you are using the starter.
Maybe it could be a good idea to make an introduction also for the starter.

Or better: split this doc in two chapters: source and starter.
On source you could say what it is and what it does.
Try to add it in the current project, explain what are the possible options and perform some example calls and show results on graphql

Next chapter should be about the starter: 
you could remember users what is a starter, explain that the plone-source is useful for fetching data ,but there is this starter that helps to bootstrap a plone-fetcher project.
Try to initialize it (like you've already done) and explain what it does

Reply:
Oh yes, that does seem like a better way of going ahead with it. I'll do a full review and in the process make these changes.